src/osd/scheduler/mClockScheduler.h

```diff
@@ -228,9 +228,53 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
   void set_config_defaults_from_profile();
 
 public: 
-  mClockScheduler(CephContext *cct, int whoami, uint32_t num_shards,
+  template<typename Rep, typename Per>
+  mClockScheduler(
+    CephContext *cct, int whoami, uint32_t num_shards,
     int shard_id, bool is_rotational, unsigned cutoff_priority,
-    MonClient *monc, bool init_perfcounter=true);
+    std::chrono::duration<Rep,Per> idle_age,
+    std::chrono::duration<Rep,Per> erase_age,
+    std::chrono::duration<Rep,Per> check_time,
```

Comment:
By using this version of the constructor do you expect configurables for the age params eventually?

Reply:
The purpose of the change is simply to make the unittest useful.  It could be useful for adding params later, but that's out of scope.